openshift_images/using_images/using-images-overview.adoc

```diff
@@ -0,0 +1,32 @@
+[id="using-images-overview"]
+= Using images overview
+include::modules/common-attributes.adoc[]
+:context: using-images-overview
+toc::[]
+
+Use the following topics to discover the different Source-to-Image (S2I),
+database, and other container images that are available for {product-title}
+users.
+
+Red Hat’s official container images are provided in the Red Hat Registry at
+link:registry.redhat.io[registry.redhat.io]. {product-title}’s supported S2I,
+database, and Jenkins images are provided in the `openshift4` repository in the
+Red Hat Quay Registry. For example,
+`quay.io/openshift-release-dev/ocp-v4.0-<address>` is the name of the OpenShift
```

Comment:
@bmcelvee is this still correct? 

@wewang58 could you confirm the content in this module (using images overview) is correct for 4.1?

Reply:
In the registry.redhat.io regsitry, jenkins images for 4.x:
regsitry.redhat.io/openshift4/ose-jenkins:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.1.4>
In quay.io, the jenkins images  are stored with digest numbers not name like this:
$oc adm release info --pullspecs quay.io/openshift-release-dev/ocp-release:4.1.4  | grep  jenkins 
  jenkins                                       quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:bec7f571b6ad293e697206786deb8194cd67d6bedf8e7eddb24290838807272a
  jenkins-agent-maven                           quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:20e18317a9284549f56814d720da33393946f8f6add15f56974015d7abb78ba1
  jenkins-agent-nodejs                          quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:98f342ba0aa7f6d012bf3f03d4a182aa92a1563ded39671b8c13f825e1348978